content/applications/hr/recruitment/offer_job_positions.rst

```diff
@@ -0,0 +1,160 @@
+
+The next step is to send an offer to the applicant. Start by selecting the desired applicant's card
+to open their applicant form.
+
+On the applicant's form, click the :guilabel:`Generate Offer` button. A :guilabel:`Generate a
+Simulation Link` pop-up window appears.
+
+Most fields are pre-populated with information from the job position. If any necessary fields are
+blank, or if any information needs to be updated, enter, or update, the relevant information in the
+corresponding fields.
+
+.. note::
+   Depending on the localization setting for the company, and which applications are installed, some
+   fields may not appear in the :guilabel:`Generate a Simulation Link` pop-up window.
+
+   For example, if the *Fleet* application is **not** installed, any fields related to vehicles do
+   **not** appear.
+
+Universal fields
+----------------
+
+The following fields appear in the :guilabel:`Generate a Simulation Link` pop-up window, regardless
+of the localization.
+
+- :guilabel:`Contract Template`: the template currently being used to populate the
+  :guilabel:`Generate a Simulation Link` pop-up window. Use the drop-down menu to select a different
+  :guilabel:`Contract Template`, if desired.
+
+  .. note::
+     To modify the template, hover over the current template name, and click the :icon:`oi-launch`
+     :guilabel:`Internal link` icon that appears to the right of the field. Make any desired
+     changes, then click :guilabel:`Save & Close`.
+
+- :guilabel:`Job Position`: the name of the :guilabel:`Job Position` being offered to the applicant.
+  The selections available in the drop-down menu correspond to the job position configured on the
+  main *Recruitment* dashboard.
+- :guilabel:`Job Title`: the selected :guilabel:`Job Position` populates this field, by default.
+  The title can be modified to suit the specific applicant's position and provide more details.
+
+  .. example::
+     An applicant is offered a marketing manager job at a shoe company, specifically for the
+     children's line.
+
+     The :guilabel:`Job Position` selected from the drop-down menu is `Marketing Manager`, and the
+     :guilabel:`Job Title` is modified for their specific responsibilities, `Marketing Manager:
+     Children's Shoes`.
+
+- :guilabel:`Department`: the :guilabel:`Department` the job position falls under.
+- :guilabel:`Contract Start Date`: the date the contract takes effect. The default date is the
+  current date. To modify the date, click on the displayed date to reveal a calendar popover window.
+  Navigate to the desired month, then click the day to select the date.
+- :guilabel:`Yearly Cost`: the annual salary being offered.
+- :guilabel:`Link Expiration Date`: the number of days the job offer is valid. The default
+  expiration date is `30` days. Modify the expiration date, if desired.
+
+Send offer
+----------
+
+Once the :guilabel:`Generate a Simulation Link` pop-up window is correct, click :guilabel:`Send By
```

Comment:
Suggestion to use "complete" instead of "correct"
```suggestion
Once the :guilabel:`Generate a Simulation Link` pop-up window is complete, click :guilabel:`Send By
```